scaling_performance/scaling_cluster_metrics.adoc

```diff
@@ -148,3 +148,38 @@ endif::openshift-enterprise[]
 If you add a new node to or remove an existing node from a Cassandra cluster,
 the data stored in the cluster rebalances across the cluster.
 ====
+
+ifdef::openshift-enterprise[]
+To scale down:
+
+. Run the following for the Cassandra node you wish to remove:
++
+----
+$ oc exec <pod> [-c <container>]
+----
+
+. Run:
++
+----
+$ nodetool decommission
+----
++
+This command can take a while to run since it copies data across the cluster.
+
```

Comment:
@jsanda: can you confirm if we can monitor the decommission progress with 'nodetool netstats -H'? that might be something we would want to mention in the notes here